service_mesh/service_mesh_day_two/ossm-auto-route.adoc

```diff
@@ -0,0 +1,66 @@
+[id="ossm-auto-route"]
+= Automatic route creation
+include::modules/ossm-document-attributes.adoc[]
+toc::[]
+
+OpenShift routes for Istio Gateways are automatically managed in {ProductName}. Every time an Istio Gateway is created, updated or deleted inside the service mesh, an OpenShift route is created, updated or deleted. 
+
+[id="ossm-auto-route-enable_{context}"]
+== Enabling Automatic Route Creation
+A {ProductName} control plane component called Istio Openshift Routing (IOR) synchronizes the gateway route. You can deploy it as part of the control plane in the {ProductName} installation process. For more information, see xref:../../service_mesh/service_mesh_install/customizing-installation-ossm.adoc#ossm-cr-gateway_customizing-installation-ossm[Istio gateway configuration].
+
+. In the `ServiceMeshControlPlane` resource, add the `ior_enabled` parameter and set it to `true`. For example, see the following resource snippet:
+
+----
+gateways:
+ istio-egressgateway:
+   autoscaleEnabled: false
+   autoscaleMin: 1
+   autoscaleMax: 5
+ istio-ingressgateway:
+   autoscaleEnabled: false
+   autoscaleMin: 1
+   autoscaleMax: 5
+   ior_enabled: true
+----
+
+[id="ossm-auto-route-subdomains_{context}"]
+== Subdomains
+
+{ProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example, `*.domain.com` are supported but not by default. Cluster administrators can refer to the OpenShift documentation for instructions on how to enable subdomains.
+
+[id="ossm-auto-route-tls_{context}"]
+== TLS
+
+If the Gateway contains a TLS section, the OpenShift Route will be configured to support TLS.
+
+For example, if the following gateway is created:
+
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: gateway1
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - www.bookinfo.com
+    - bookinfo.example.com
+----
+
+Then, the following OpenShift Routes are created automatically. You can check that the routes are created with the following command. 
+
+----
+$ oc -n istio-system get routes
```

Comment:
nit: `istio-system` changes depending on the control plane namespace. Not sure if that matters.

Reply:
I changed the istio system to a variable for your control plane. Is that right?

Reply:
Works for me. I just know that in other places, we say, "your control plane namespace, for example `istio-system` or something similar"